x/staging/subspaces/types/msgs.go

```diff
@@ -32,11 +32,11 @@ func (msg MsgCreateSubspace) ValidateBasic() error {
 	}
 
 	if !IsValidSubspace(msg.SubspaceID) {
-		return sdkerrors.Wrap(ErrInvalidSubspaceID, "subspace id must be a valid SHA-256 hash")
+		return ErrInvalidSubspaceID
```

Comment:
As above, we can wrap this
```suggestion
		return sdkerrors.Wrap(ErrInvalidSubspace, ErrInvalidSubspaceID)
```